src/recert.rs

```diff
@@ -16,7 +22,41 @@ pub(crate) mod timing;
 pub(crate) async fn run(recert_config: &RecertConfig, cluster_crypto: &mut ClusterCryptoObjects) -> Result<RunTimes> {
     ensure_openssl_version().context("checking openssl version compatibility")?;
 
-    let in_memory_etcd_client = get_etcd_endpoint(recert_config).await?;
+    let resource_transformers = get_resource_transformers(recert_config).await?;
+    let mut in_memory_etcd_client = get_etcd_endpoint(recert_config, resource_transformers.clone()).await?;
+
+    if resource_transformers.is_some() {
+        #[allow(clippy::unwrap_used)]
+        let mut resource_to_prefix_transformers: HashMap<_, _> = resource_transformers.unwrap().resource_to_prefix_transformers;
+
+        let apiserver_location = K8sResourceLocation::new(Some("openshift-apiserver"), "Secret", "encryption-config", "v1");
+        if let Some(apiserver_resource_transformers) =
+            get_openshift_resource_transformers(&in_memory_etcd_client, &apiserver_location).await?
+        {
+            resource_to_prefix_transformers = resource_to_prefix_transformers
+                .into_iter()
+                .chain(apiserver_resource_transformers.resource_to_prefix_transformers)
```

Comment:
Since the hashmap is already `mut`, you can `.extend` it. If you want to use `chain` (which I prefer), it's better it you remove `mut` and redeclare (shadowing) `resource_to_prefix_transformers` with another `let resource_to_prefix_transformers = ...`

Reply:
oh nice, I agree with avoiding `mut`, let's use `shadowing` instead. I'll update this  :+1: